### PR TITLE
ECEP-113: Install & Style News (ECEP, Texascale)

### PR DESCRIPTION
**Pull request by wesleyboar**
### Overview / Changes

- Clean Texascale custom settings.
- Add ECEP custom settings for blog.
- (Noop) Comments for ECEP site CSS.
- (Tangential) Update Texascale CSS based on Core-Styles changes.

### Related

- [ECEP-113](https://jira.tacc.utexas.edu/browse/ECEP-113)
- requires & required by https://github.com/TACC/Core-CMS/pull/466
- requires https://github.com/TACC/Core-Styles/pull/16

### Screenshots & Testing

See https://github.com/TACC/Core-CMS/pull/466.

### CODE REVIEW

File: texascale-org/templates/snippets/core-gl-19-support-align-center.html [hunk 1]
```diff
@@ -1,7 +0,0 @@
-<style id="core-gl-19--add">
-    .align-center {
-        display: block;
-        margin-left: auto;
-        margin-right: auto;
-    }
-</style>
```
Comment (wesleyboar) [inline texascale-org/templates/snippets/core-gl-19-support-align-center.html:7]: (moved to [TACC/Core-CMS:/taccsite_cms/static/site_cms/css/src/_imports/components/django.cms.picture.css](https://github.com/TACC/Core-CMS/blob/task/ECEP-113-support-news/taccsite_cms/static/site_cms/css/src/_imports/components/django.cms.picture.css))

File: texascale-org/static/texascale-org/css/src/_imports/objects/o-offset-content.css [hunk 2]
```diff
@@ -1,5 +1,18 @@
...
+  .figure:is(:--o-offset-content) .figure-caption {
+    padding-right: unset;
+  }
+}
+@media only screen and (--medium-and-above) and (--max-wide-and-below) {
+  .o-offset-content--left,
+  .o-offset-content--right {
+    max-width: 50%;
+  }
+}
```
Comment (wesleyboar) [inline texascale-org/static/texascale-org/css/src/_imports/objects/o-offset-content.css:18]: **Explanation.** Update styles based on [TACC/Core-Styles#16:`/source/_imports/objects/o-offset-content.css`](https://github.com/TACC/Core-Styles/pull/16/files#diff-4568efa6476eb9b665d81688a5b81e8d3b825de2d6503f47db5be6f749c0253b) changes.

<details>

- Use new customs elector `:--o-offset-content`.
- Retain Texascale's`--offset-distance` value.
- Do **not** use Core's newly-added `padding-right` value.
- Do **not** use Core's updated `max-width` value.

</details>

File: ecep-cms/settings_custom.py [hunk 3]
```diff
@@ -79,3 +79,62 @@
...
 
 INCLUDES_CORE_PORTAL = True
+
+########################
+# TACC: NEWS/BLOG
+########################
+
+# TACC settings
+TACC_BLOG_SHOW_CATEGORIES = False
+TACC_BLOG_SHOW_TAGS = False
```
Comment (wesleyboar) [inline ecep-cms/settings_custom.py:89]: **Note.** ECEP does not use categories nor tags, and has no immediate interest in doing so. Other projects could be similar.

**Review by wesleyboar — COMMENTED** [review]
notes for reviewers

File: ecep-cms/settings_custom.py (hunk 3 above)
Comment (taoteg) [reply ecep-cms/settings_custom.py:89]: I like the flexibility!

**Review by taoteg — APPROVED** [review]
LGTM.

**Comment by wesleyboar** [comment]
All related PR have been approved. I merge now.

**Pull request merged**